methods_and_recursion/bottles_of_beer.rb

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+
+def sing_a_verse(number1, number2)
+  puts "#{number1} bottle#{'s' unless number1 == 1} of beer on the wall,"\
+       " #{number1} bottle#{'s' unless number1 == 1} of beer."
+  puts "Take one down and pass it around,"\
+       " #{number2} bottle#{'s' unless number2 == 1} of beer on the wall."
+  puts
```

Comment:
Best practice is to use `puts ''`.  A `puts` by itself makes a future developer wonder if you intended to not having anything after the puts.  Using `puts ''` makes it more explicit.